groupAndroidPackages.json

```
@@ -300,6 +300,12 @@
         "^com\\.squareup\\.leakcanary:"
       ]
     },
+    {
+      "groupName": "Lottie",
+      "matchPackagePatterns": [
+        "^com\\.airbnb\\.android:lottie-"
```

This only matches against `com.airbnb.android:lottie-compose:` and doesn't against `com.airbnb.android:lottie:`.

```suggestion
        "^com\\.airbnb\\.android:lottie-?"
```

---

Thanks!
Fixed it.